pallets/vector/Cargo.toml

```diff
@@ -20,6 +20,9 @@ sp-runtime = { workspace = true, default-features = false }
 sp-std = { workspace = true, default-features = false }
 sp-core = { workspace = true, default-features = false }
 frame-benchmarking = { workspace = true, default-features = false, optional = true }
+# TODO is there a release version?
```

Comment:
The `4.0.0-rc1` tag should be out in the next few days, which you can use. For now, I'd recommend pinning to a specific commit.